### ISSUE TITLE
I'll choose love it.....

### ISSUE THREAD

**Issue by PySimpleGUI** [spdkils/python_tutoring#1]
I like the wide range of choices... I'll take the love it choice ![image](https://user-images.githubusercontent.com/46163555/185817376-7e8bb276-0542-4f7e-8ecf-cee64825171c.png)

I don't know what a gcode file is so I don't have any to open.  Can you post one?  Maybe drop a screenshot in your readme since you've gone to the trouble of making a GUI.

![image](https://user-images.githubusercontent.com/46163555/185817463-4eaba813-d9c8-4cfb-99de-ac73333bbe3b.png)

I like the random theme, by design, by **choice**  (like your readme)... literally by choice

https://github.com/spdkils/python_tutoring/blob/52f5d5d34912c859b3282ff2344d3b6eb231f02e/wam_gcode/wam_decode.py#L203

**Comment by spdkils** [comment]
I just added a gcode file. GCODE is the standard for computer controlled machining. It's used for 3d printers, milling machines, as well as Abrasive Water Jet (AWJ) my use case.

I included an example in this 2d cut that shows exactly why I built this tool. Some outside cuts will separate the materiel from the base. Most software has the ability to detect closed paths, and add tabs to the material, as well as other options. This particular product does not behave the way it should in certain cuts.

As such this was created to allow my wife/friends to use the water-jet, and quickly preview/fix cut order for their small business.

There have been several updates in the last month, quick file choosing via listbox, search via regex (to fix/deal with case sensitivity, and other enhancements.)

![image](https://user-images.githubusercontent.com/11712349/192323269-9995fe00-df46-48ef-8019-f119cd078897.png)
